tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -565,6 +589,7 @@ def with_integration_base_java_and_normalization(context: PipelineContext, build
     ]
 
     dbt_adapter_package = DESTINATION_SPECIFIC_NORMALIZATION_ADAPTER_MAPPING.get(context.connector.technical_name, "dbt-bigquery==1.0.0")
```

Comment:
@alafanechere should these two lines actually raise an exception? I.e. if the build isn't configured to support this connector, we would fail loudly

Reply:
Yes, should all destination have a custom adapter, can we fallback to a default python package, or should adapter be optionally installed?

Reply:
normalization is somehow installing a couple packages by default:
```
$ docker run -it --entrypoint=bash airbyte/normalization:dev
proot@ef8b6bf1d4ac:/airbyte# pip list | grep dbt
dbt-bigquery               1.0.0
dbt-core                   1.0.0
dbt-extractor              0.4.0
dbt-postgres               1.0.0
dbt-redshift               1.0.0
dbt-snowflake              1.0.0
```

so I'll just copy those values :shrug:

Reply:
also switched everything to just use `[...]` instead of `.get`.